types/google-apps-script/test/google-apps-script-tests.ts

```diff
@@ -776,3 +776,15 @@ const sheetCellImage = () => {
     cellImage.getContentUrl();
     cellImage.getUrl();
 };
+
+// Blob test
+const blob = () => {
+    // $ExpectType Blob
+    const blob = Utilities.newBlob('content', 'type');
```

Comment:
It may be better for the `contentType` argument value to be valid than meaningless.
Example:

```js
const blob = Utilities.newBlob('content', 'application/json');
```

Copy from https://developers.google.com/apps-script/reference/utilities/utilities#newBlob(String,String)